### ISSUE TITLE
Sync for passwords stops working after a month and cookies are deleted

### ISSUE THREAD

**Issue by JackKenney** [brave/brave-browser#13739]
## Description 

For the second or third time on Brave has stopped syncing my passwords. The "Sync Everything" toggle is turned off and all the sync toggles are off. 

Checking all of them and refreshing the settings screen returns them all to toggled off.

## Steps to Reproduce

As far as I can tell
   1. Fresh install Brave
   2. Add brave to existing sync chain
   3. Notice it works fine for about a month or two
   4. One day it will stop syncing passwords and all the cookies will be deleted

Bookmarks and other settings appear to continue to sync or at least stay added to local bookmarks list.

## Reproduces how often: 

Intermittent issue, reporting after second time it's happened

## Brave version (brave://version info)

Brave | 1.18.77 Chromium: 87.0.4280.101 (Official Build) (64-bit)
-- | --
Revision | 9407c80213cda69c2b7abcb4fa8e3f74488f4956-refs/branch-heads/4280@{#1807}
OS | Linux
JavaScript | V8 8.7.220.29
Flash | (Disabled)
User Agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.101 Safari/537.36
Command Line | /opt/brave.com/brave/brave --enable-dom-distiller --disable-domain-reliability --no-pings --extension-content-verification=enforce_strict --extensions-install-verification=enforce --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --variations-server-url=https://variations.brave.com/seed --enable-features=PrefetchPrivacyChanges,DnsOverHttps,LegacyTLSEnforced,PasswordImport,ReducedReferrerGranularity,AutoupgradeMixedContent,WebUIDarkMode --disable-features=AutofillServerCommunication,TextFragmentAnchor,PrivacySettingsRedesign,PasswordCheck,NetworkTimeServiceQuerying,SmsReceiver,NotificationTriggers,AllowPopupsDuringPageUnload,AutofillEnableAccountWalletStorage,TabHoverCards,SignedExchangeSubresourcePrefetch,SafeBrowsingEnhancedProtection,IdleDetection --flag-switches-begin --flag-switches-end
Executable Path | /opt/brave.com/brave/brave
Profile Path | /home/<user>/.config/BraveSoftware/Brave-Browser/Default

## Version/Channel Information:

- Can you reproduce this issue with the current release?
I upgraded last time and it did not fix the issue. Can try upgrading again.
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?: Unknown
- Does the issue resolve itself when disabling Brave Rewards?: Unknown
- Is the issue reproducible on the latest version of Chrome?: Unknown

**Comment by jsecretan** [comment]
Sorry for the inconvenience @JackKenney I had completely missed this.  If you are still experiencing this, do you think you could upload your brave://sync-internals to this form:

https://upload-support.brave.com/sync

That would help our team to debug this issue

**Comment by bsclifton** [comment]
I remember a bug with this being fixed a while back - @JackKenney please let us know if you're still experiencing this

**Comment by JackKenney** [comment]
Yeah this still seems to be happening with one of my sync chains. 
I started another chain because it was too much of an issue, but here is the support ID from the link above:
419a5ca8-f673-4308-b358-99f5536393bc

**Comment by jsecretan** [comment]
Great, thanks @JackKenney
